django/utils/http.py

```diff
@@ -110,16 +110,18 @@ def urlencode(query, doseq=False):
 
 def cookie_date(epoch_seconds=None):
     """
-    Format the time to ensure compatibility with Netscape's cookie standard.
+    Format the time
```

Comment:
add: "according to...."

This function is also documented at `docs/ref/utils.txt`. The documentation should be updated, including a ``.. versionchanged:: 2.1` annotation. A release note should be added in `docs/releases/2.1.txt` under backwards-incompatible changes.

Reply:
Looks like `cookie_date` is same to `http_date`, and should be replaced by it. Haven't noticed it before.

Reply:
I need mark it as deprecated? I think i couldn't just remove it

Reply:
Deprecation could be a separate ticket/task. For now, just update the documentation to indicate how it has changed.